src/lwc/edaSetupHealthCheckRun/edaSetupHealthCheckRun.html

```diff
@@ -0,0 +1,37 @@
+<template>
+    <lightning-card title={labelReference.stgEDASettingsHealthCheckTitle}>
+        <div class="slds-grid slds-wrap">
+            <div class="slds-col 
+                        slds-size--1-of-1 
+                        slds-p-horizontal_small 
+                        slds-p-vertical_small
+                        slds-text-body_regular">
+                <p>{labelReference.stgHealthCheckDescription}</p>
+            </div>
+
+            <div class="slds-col 
+                        slds-size--1-of-1 
+                        slds-medium-size--1-of-2 
+                        slds-large-size--1-of-2 
+                        slds-text-align_left
+                        slds-text-body_small 
+                        slds-p-horizontal_small
+                        slds-p-vertical_small">
+                {lastRunDate}
+            </div>
+            <div class="slds-col 
+                        slds-size--1-of-1 
+                        slds-medium-size--1-of-2 
+                        slds-large-size--1-of-2 
+                        slds-p-horizontal_small
+                        slds-p-vertical_small">
+                <lightning-button
+                    class="slds-float_left"
```

Comment:
Avoid using non-slds styling if at all possible. SLDS does a decent amount of sorcery that we don't want to interfere with.